.github/workflows/ci-post-commit-analyzer-run.py

```diff
@@ -0,0 +1,34 @@
+import json
+import multiprocessing
+import os
+import re
+import subprocess
+import sys
+
+
+def run_analyzer(data):
+    os.chdir(data["directory"])
+    command = (
+        data["command"] 
+        + f" --analyze --analyzer-output html -o analyzer-results -Xclang -analyzer-config -Xclang max-nodes=75000"
+    )
+    print(command)
+    subprocess.run(command, shell=True, check=True)
+
+
+def pool_error(e):
+    print("Error analyzing file:", e)
```

Comment:
```suggestion
    print("Error analyzing file: ", e)
```

Reply:
Doesn't the print function automatically put a space between arguments?

Reply:
Oh crap nvm I forgot about that lol.